dags/veda_data_pipeline/utils/s3_discovery.py

```diff
@@ -221,10 +227,10 @@ def s3_discovery_handler(event, chunk_size=2800, role_arn=None, bucket_output=No
         raise ValueError(f"No files discovered at bucket: {bucket}, prefix: {prefix}")
 
     # out of convenience, we might not always want to explicitly define assets
-    if assets is not None:
+    if assets and len(assets) > 1:
```

Comment:
is assets still a dict at this point or has this variable been transformed for the event? Otherwise `len(assets.keys()) > 1`

Reply:
you're right, it is still a dict, thanks for catching that!